forge-gui/res/adventure/Shandalar/world/points_of_interest.json

```diff
@@ -1850,6 +1850,15 @@
 	"sprite": "BasilicaSmall",
 	"map": "maps/map/phyrexian_w1.tmx"
 },
+{
+	"name": "GitaxianLab",
```

Comment:
Crash when entering the dungeon on my end. This is the error 
![image](https://user-images.githubusercontent.com/67333662/227706146-8477de16-687c-4081-ad66-ed3a7e9af708.png)

Reply:
Should be fixed - it was looking for booster.tx in my test environment instead of the filesystem itself for some reason.